sort_by.rb

```diff
@@ -0,0 +1,47 @@
+# Defining  bubble_ Sort Defination
+def bubble_sort(array)
+	n = array.length
+	swapped = true	
+		while swapped do
+		swapped = false
+		(n - 1).times do |i|
+			if array[i] > array[i + 1]
+			array[i], array[i + 1] = array[i + 1], array[i]
+			swapped = true
+			end
+		end
+	end
+	puts "This is the sorted list #{array}"
```

Comment:
Let it just output only the array no strings

Reply:
@kobiyoyo Could you just explain it to me why DO i need to do the following , its not mentioned in the criteria that we cant have a string as out put.
I have made the changes but Still want to know what made you say that I need to just print array without strings ?